### ISSUE TITLE
Improving Readiblity

### ISSUE THREAD

**Issue by deveshio** [codervivek5/VigyBag#923]
I'd like to suggest that project can have a better organised form 
- we can add all HTML files in one folder except index 
- we can keep all text file in one folder 
- we can have log file in one place 
- we can have js file in a single script folder 
- we can have CSS in a style folder 
this will make project more readable and more understandable for newcontributors

**Comment by deveshio** [comment]
I'd like to work on this project

**Comment by codervivek5** [comment]
before make any PR...kindly show me file structure which you will arrange

**Comment by deveshio** [comment]
Sure , i am working on it

**Comment by shnwz-solves** [comment]
Hey PA, I want to contribute, please assign this issue to me. Thankyou!

**Comment by codervivek5** [comment]
start working but before create PR just contact with me

**Comment by Hrusikesh001** [comment]
Hi @codervivek5  ,

I'm Hrusikesh, a Full-Stack Developer. I want to contribute.

Could you please assign this issue to me under OSCI?

Thank you! 😊

**Comment by codervivek5** [comment]
> Hi [@codervivek5](https://github.com/codervivek5) ,
> 
> I'm Hrusikesh, a Full-Stack Developer. I want to contribute.
> 
> Could you please assign this issue to me under OSCI?
> 
> Thank you! 😊

Yes I can assign but your contribution will be count from 16 august so I suggest you wait till then and again drop a recall  comment then i'll assign you

**Comment by Hrusikesh001** [comment]
Okay, Thank You @codervivek5
